tests/t235_report_field1.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env python
+
+from runtest import TestBase
+
+class TestCase(TestBase):
+    def __init__(self):
+        TestBase.__init__(self, 'sort', """
+  Total time       Calls   Function
+  ==========  ==========   ====================
+   11.173 ms           1   main
+   10.467 ms           1   bar
+   10.297 ms           1   usleep
+  207.139 us           2   foo
+  204.033 us           6   loop
+    0.763 us           1   __monstartup
+    0.299 us           1   __cxa_atexit
+""")
+
+    def prepare(self):
+	self.subcmd = 'record'
+	return self.runcmd()
+
+    def setup(self):
+	self.subcmd = 'report'
+        self.option = '-f total,call'
```

Comment:
Please check indentation especially when SPACE and TAB are intermixed.

Reply:
sorry. ok